src/components/HomeTiles.tsx

```diff
@@ -36,7 +35,7 @@ export default function HomeTiles(props:HomeTilesProperties) {
         </Typography>
       </CardContent>
       <CardActions>
-        <Button size="small" onClick={() => navigate((props.route), {state: data})}>Total Published Entries: {props.entrycount}</Button>
+        <Button size="small" onClick={() =>  window.open(props.route ,'_blank', 'rel=noopener noreferrer')}>Total Published Entries: {props.entrycount}</Button>
```

Comment:
<Button size="small" onClick={() => window.open(props.route ,'_blank', 'rel=noopener noreferrer')}>Total Published Entries: {props.entrycount}</Button>
```typescript
<Button size="small" onClick={() => window.open(props.route, 'rel=noopener noreferrer')}>Total Published Entries: {props.entrycount}</Button>
``` 

That fixes it my end.